test/caesium/binding_test.clj

```diff
@@ -83,9 +83,12 @@
 (defn check-const-method
   "Check a method binding a const fn."
   [^Method method]
-  (let [rtype (.getGenericReturnType method)]
+  (let [rtype (.getGenericReturnType method)
+        int-funcs #{"sodium_init" "crypto_pwhash_argon2i_alg_argon2i13"
+                    "crypto_pwhash_alg_default" "crypto_pwhash_alg_argon2i13"
+		    "crypto_pwhash_argon2id_alg_argon2id13"}]
```

Comment:
Just checking why this change is here; if I understand correctly, the reason is because this is the first time that we've implemented alg const fns that return int. All of the other const fns so far are longs or strs. Is that accurate?

Reply:
that is correctly. the other fns listed here specifically return int to identify the underlying algo. libsodium implementation has a "case" statement that checks the value before executing the correct function.